### ISSUE TITLE
Improvement for the centrifuge

### ISSUE THREAD

**Issue by TimBomBom** [studio-minus/ppg-bugs-and-requests#2004]
Like the centrifuge but it has a separate chamber where each low-concentration component gets filtered into after each cycle instead of being destroyed. This simple change will allow refinement of liquids much easier and less frustrating. In real centrifuges, the liquids aren't destroyed until the densest is left, but they are separated.

With this change, it will finally be possible to refine an exact component from a mixture without going through extremely tedious processes until the desired component is finally isolated. You could just let the desired part get into the secondary tank and then take it out of there into storage.

![image](https://user-images.githubusercontent.com/84190142/186041305-36c9706b-3d0a-44b8-9965-8cbd5c172907.png)

Let's have an example: I have a pink syringe and I want to isolate/extract pure circulation poison. This is how the machine would be used to do so:

![image](https://user-images.githubusercontent.com/84190142/186041310-a2ac5b3a-08fa-4887-aeba-49761574244e.png)

I wrote this request because I like isolating liquids and playing around with pink syringes but it is an absolute pain to isolate a liquid when there are other components of higher concentration in the mixture (namely inert pink fluid)

**Comment by DindinYT37** [comment]
Duplicate of #2003

**Comment by DindinYT37** [comment]
Still a duplicate of #2003, even if it doesn't have the tag.

**Comment by TimBomBom** [comment]
fair enough. Can you delete this one and give the tag and assignment to #2003 then?

**Comment by DindinYT37** [comment]
> fair enough. Can you delete this one and give the tag and assignment to #2003 then?

This is not my repository? I'm just telling you this is a duplicate, zooi (the Developer and Owner of this repository) will probably do it when he checks it though. So basically all you have to do now is wait.

**Comment by TimBomBom** [comment]
oh I thought you were like a mod or smth ok

**Comment by DindinYT37** [comment]
> oh I thought you were like a mod or smth ok

It's just a generic GitHub thing to tell others when an issue is a duplicate etc.
